README.md

```diff
@@ -1,2 +1,17 @@
 # HW01
-Edit the readme
+### Andrea Watson
+
+Hi! I'm a rising 5th year PhD candidate in the Committee on Microbiology. Here is my ~~not very professional~~ twitter: **[\@awatsontxt](https://twitter.com/awatsontxt)**
+
+Some facts about me:
+
+  * I am Canadian
+  * I study gut microbes
+  * I'm adopting a cat in 6 days, who is:
+      1. fluffy
+      2. white
+      3. small
+  
+Here is a picture of my future cat:
+
+![(Her name is Willow)](https://i.imgur.com/8htre7V.jpg)
```

Comment:
She is so adorable. Where did you find her? It's a good name as well. I never thought to name a cat with a tree's name.